ding/obs_model/tests/test_curl_obs_model.py

```diff
@@ -0,0 +1,49 @@
+import pytest
+import torch
+import torch.nn as nn
+from ding.obs_model import *
+# from ding.obs_model import CurlObsModel
+# from ding.obs_model import Encoder
+from torch.autograd import Variable
+
+# encoder = nn.Linear(32, 50)
+# encoder_target = nn.Linear(32, 50)
+
+def test_curl_compute_logits():
+    curl = CurlObsModel(CurlObsModel.default_config(), None)
+    z_anc = torch.randn(4, 50)
+    z_pos = torch.randn(4, 50)
+    logits = curl.compute_logits(z_anc, z_pos)
+    assert logits.shape == (4, 4)
+    print('end')
+
+def test_curl_encode():
+    curl = CurlObsModel(CurlObsModel.default_config(), None)
+    # x : shape : [B, C, H, W] [batch_size, frame_stack, height, width]
+    x = torch.FloatTensor(64, 3 * curl.cfg.frame_stack, 84, 84)
```

Comment:
modify tensor creation and know more about `torch.randn, torch.rand, torch.randint`